templates/CRM/Contact/Form/Task/Map/OpenStreetMaps.tpl

```diff
@@ -32,7 +32,7 @@
 {/if}
 {assign var=defaultZoom value=12}
 {literal}
-<script src="http://openlayers.org/api/OpenLayers.js" type="text/javascript"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/openlayers/2.13.1/OpenLayers.js" type="text/javascript"></script>
```

Comment:
is there any reason to specify protocol here? Seems like it might be better to just start with `//`